### PR TITLE
EIP („Ground Zero“), Draft

### PR DESCRIPTION
**Pull request by ProphetZX**
This EIP suggest protocol changes, see details in the EIP description.

### CODE REVIEW

**Comment by g11tech** [comment]
Closed the PR as this is either a parody PR at best or a spam at worst

**Comment by Pandapip1** [comment]
> Closed the PR as this is either a parody PR at best or a spam at worst

I believe this user to be a bot that spams the EIP repo. @ProphetZX if you would like to indicate that you are not, please say "I am not a robot." Thanks!

**Comment by ProphetZX** [comment]
I am definitely not a bot and this is not spam, but a serious proposal draft, in light of the very high transaction and gas costs in the current setting. If you’d like to, we can also switch to a different channel to discuss these issues.

**Comment by YaroslavPysanskyi** [comment]
I am not a robot.

**Comment by Pandapip1** [comment]
> I am definitely not a bot and this is not spam, but a serious proposal draft, in light of the very high transaction and gas costs in the current setting. If you’d like to, we can also switch to a different channel to discuss these issues.

Just to clear up any misunderstanding you may have - a large part (perhaps the largest part) of why Ethereum exists is because of its decentralized, unpermissioned nature. Both Ethereum and IPFS are useful, and I agree that too much is being done on-chain that would likely be better suited to a SMPC. However, that doesn't mean that Ethereum is useless, simply that it has its own niche in which its unpermissioned nature is crucial.

File: config/.codespell-whitelist [hunk 1]
```diff
@@ -1,16 +1 @@
...
-ist
-iam
-espace
-acn
-ende
-sting
-complies
-shiping
-furter
+MRnew
```
Comment (Pandapip1) [inline config/.codespell-whitelist:1]: I don't know why you wold be overriding this file

File: config/eip-editors.yml
Comment (Pandapip1) [inline config/eip-editors.yml:1]: This looks like it was maliciously intended. @ProphetZX, can you explain why you were touching this file at all?

**Comment by ProphetZX** [comment]
Ok, thanks for the explanations. Do you mind if I ask whether the community was satisfied with the permissonless nature of Ethereum over the past years?On 17 Jul 2023, at 16:54, Gavin John ***@***.***> wrote:﻿

I am definitely not a bot and this is not spam, but a serious proposal draft, in light of the very high transaction and gas costs in the current setting. If you’d like to, we can also switch to a different channel to discuss these issues.

Just to clear up any misunderstanding you may have - a large part (perhaps the largest part) of why Ethereum exists is because of its decentralized, unpermissioned nature. Both Ethereum and IPFS are useful, and I agree that too much is being done on-chain that would likely be better suited to a SMPC. However, that doesn't mean that Ethereum is useless, simply that it has its own niche in which its unpermissioned nature is crucial.

File: config/.codespell-whitelist (hunk 1 above)
Comment (ProphetZX) [reply config/.codespell-whitelist:1]: This is related to the EIP draft and I thought to have spotted some malicious entries there.

**Comment by ProphetZX** [comment]
> > Do you mind if I ask whether the community was satisfied with the permissonless nature of Ethereum over the past years?
> 
> That's like asking if Google search's users are happy that the results they get are ordered. It's the reason Ethereum exists in the first place.

I cannot speak for you, but I know a ton of people who despise Google’s sorting of search results (as well as their Ireland extension and policies). ;)

Discussing with some people which are active in the Ethereum community, I got the expression that most were quite unhappy with the problems the permissonless nature of Ethereum creates.

**Comment by Pandapip1** [comment]
> Discussing with some people which are active in the Ethereum community, I got the expression that most were quite unhappy with the problems the permissonless nature of Ethereum creates.

I'm curious - who did you speak to?

File: config/eip-editors.yml
Comment (ProphetZX) [reply config/eip-editors.yml:1]: No malicious intend, simply a governance and security improvement suggestion.

File: EIPS/eip-0.md
Comment (ProphetZX) [inline EIPS/eip-0.md:1]: May I ask you to please reopen the PR and review it, thx.

**Review by ProphetZX — COMMENTED** [review]
May I kindly ask to review these changes and merge the PR, thx.

**Pull request closed without merging**